### ISSUE TITLE
My friend cannot connect to a server because Arma 3 says CBA is not detected even though it is installed

### ISSUE THREAD

**Issue by BroMcBrostein** [CBATeam/CBA_A3#1642]
Hello!

As the title says my friend cannot connect to a server because Arma 3 says CBA is not installed even though it is. We have tried reinstalling CBA through Steam Workshop and manually multiple times but it still doesn't seem to work. CBA is in the same folder as all the other mods.
![Arma3](https://github.com/CBATeam/CBA_A3/assets/158459478/854e7d4b-d3eb-494c-95bb-d33b3a89af0d)

**Comment by BroMcBrostein** [comment]
Is this what you were asking for? I'm not 100% if I understood you correctly. Also I'm a finn too if you want to continue in finnish to make things clearer :D

![Arma3 1](https://github.com/CBATeam/CBA_A3/assets/158459478/16e7d404-ce32-44e2-8a3d-df9429b4f457)

**Comment by rautamiekka** [comment]
The preset file, I should say (same thing, "preset file" is just specific to 'Arma 3'), that he's using, and one you should be. **Never share more than 1-2 mods with 1-2 other ppl without the preset file, it'll save your life.**

1. Open the launcher, go to the 'MODS' tab.
2. Top right corner of the window, right above the search bar, is a 'PRESET' button. Click it, then if the preset ain't saved yet, save it, then click the 2 arrows button in the same line as the preset's name, which saves a HTML file.
3. Attach the HTML file.

**Comment by BroMcBrostein** [comment]
This platform doesn't let me share a html file so I converted it to PDF. Is this what you wanted to see? 
[Arma 3 preset.pdf](https://github.com/CBATeam/CBA_A3/files/14151277/Arma.3.preset.pdf)

**Comment by rautamiekka** [comment]
Close enough.

Can't see anything wrong with the items that could be related, so I'm thinking the CBA install may be corrupted.

1st check `%APPDATA%\..\Local\Arma 3\Arma3_*.rpt`, minding the timestamp.

**Comment by BroMcBrostein** [comment]
How do I know if the file is corrupted?

**Comment by rautamiekka** [comment]
The log will say something to indicate something ain't right, most likely finding syntax errors that didn't exist in an intact file, but that's just 1 possible example.

**Comment by BroMcBrostein** [comment]
I went through one of his .rpt files and found an error C0000005 ACCESS_VIOLATION and already started fixing it. It seems very likely that CBA has nothing to do with his problem but thank you anyway for helping.

While troubleshooting I realized that Arma says CBA is not detected for me too just like it does in the picture that I included in the opening post of this thread. Only thing that's different is that I am able to join to the server. Idk what's that all about?
